vividus-tests/src/main/resources/story/integration/RestApiStepsTests.story

```diff
@@ -7,7 +7,7 @@ Lifecycle:
 Before:
 Scope: STORY
 When I initialize the story variable `request-body` with value `{
-    "string": "string",
+    "string": "<, ' symbols codes are &lt;, &#39;",
```

Comment:
also please add `&#58;&#34;` (having these symbols it would be easier to notice if something is broken)